include/ripples/graph.h

```
@@ -546,11 +585,10 @@ class Graph {
     return G;
   }
 
-  edge_type **csr_index() const { return index; }
+  edge_pointer_t *csr_index() const { return index; }
 
-  edge_type *csr_edges() const { return edges; }
+  edge_pointer_t csr_edges() const { return edges; }
 
- private:
   template <typename FStream>
   void load_binary(FStream &FS) {
```

This function may not work with the offset pointer. We may want to throw an error if this function is called when Metall is enabled.

---

@neffrw Add check